gameboard/setupships.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="setupships.css">
+    <title>Document</title>
```

Comment:
Wouldn't it be easier, to control the HTML page with JS file rather than a few views?